Mdkessler patch 12 (#277)
Benjamin-Lee/deep-rules: content/11.interpretation.md

## Tip 9: Don't over-interpret predictions {#interpretation}

Once we have trained an accurate deep learning model, we often want to use it to deduce relationships and inform scientific findings.
However, in doing this, we need to be careful to correctly interpret the model's predictions.

> Given that deep learning models can be difficult to interpret intuitively, there is often a temptation to anthropomorphize the models, and to overinterpret the predictions in indulgent and/or inaccurate ways.

Reviewer: Note that this line the focus of #268, which takes offense at the use of "anthropomorphize" in this context. Maybe we should just remove the phrase and replace it with what you wrote:

```suggestion
Given that deep learning models can be difficult to interpret intuitively, there is often a temptation to overinterpret the predictions in indulgent and/or inaccurate ways.
```

Unsurprisingly, the neural network model achieved the best predictive accuracy.
However, after fitting a rule-based model in order to better understand the relationships inherent to their data, the authors discovered that the hospital data implied the rule `HasAsthma(x) => LowerRisk(x)`.
This rule contradicts medical understanding, as having asthma doesn't make pneumonia better!
Nonetheless, this rule was supported by the data, as pneumonia patients with a history of asthma tended to receive more aggressive care.
The neural network had therefore also learned to make predictions according to this rule despite the fact that it has nothing to do with causality or mechanism.
Guiding treatment decisions according to the predictions of the neural network would have been disastrous, even though the neural network had high predictive accuracy.

To trust deep learning models, we must combine knowledge of the training data ([Tip 4](#know-your-problem)) with inspection of the model ([Tip 8](#blackbox)).

> To move beyond fitting predictive models and towards the building of an understanding that can inform scientific deduction, we suggest working to disanetangle a model's internal logic by comparing data domains where your model succeeds to those in which they fail. And again, take care not to overinterpret or anthropomorphize your models.

Reviewer 2: ```suggestion
To move beyond fitting predictive models and towards the building of an understanding that can inform scientific deduction, we suggest working to disentangle a model's internal logic by comparing data domains where models succeeds to those in which they fail.
And again, take care not to overinterpret or anthropomorphize models.
```

Reviewer 2: Last like says anthropomorphize again, but fixed the other elements

However, after fitting a rule-based model in order to better understand the relationships inherent to their data, the authors discovered that the hospital data implied the rule `HasAsthma(x) => LowerRisk(x)`.
This rule contradicts medical understanding, as having asthma doesn't make pneumonia better!
Nonetheless, this rule was supported by the data, as pneumonia patients with a history of asthma tended to receive more aggressive care.
The neural network had therefore also learned to make predictions according to this rule despite the fact that it has nothing to do with causality or mechanism.
Guiding treatment decisions according to the predictions of the neural network would have been disastrous, even though the neural network had high predictive accuracy.

To trust deep learning models, we must combine knowledge of the training data ([Tip 4](#know-your-problem)) with inspection of the model ([Tip 8](#blackbox)).
To move beyond fitting predictive models and towards the building of an understanding that can inform scientific deduction, we suggest working to disentangle a model's internal logic by comparing data domains where models succeeds to those in which they fail.

> And again, take care not to overinterpret or anthropomorphize models.

Reviewer: ```suggestion
By doing so, we can avoid overinterpreting models and view them for what they are: complex statistical models trained on high dimensional data.
```

Author: @Benjamin-Lee I like your change.

Here's another alternative for consideration...just to throw it out there in case any of it appeals to anyone.

```suggestion
By doing so, we can avoid over interpreting these highly complex and frequently caveated statistical models.
```

Reviewer 2: I think Ben's works well. Shall we go with that?
